Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -358,6 +358,22 @@ internal static IEnumerator WaitForEnterKey()
             }
         }
 
+        /// <summary>
+        /// Sometimes it take a few frames for inputs raised via InputSystem.OnInput*
```

Comment:
Huh, that seems concerning. Do you have any ideas why there might be a few frames of lag?

Reply:
No, I wasn't sure how to debug that. Any ideas?